docs/manual/nullness-checker.tex

```diff
@@ -96,6 +96,20 @@
   See Section~\ref{nullness-arrays} for a discussion.
   % TODO: this option is temporary and the sound option
   % should become the default.
+
+\item
+  \label{collection-object-parameters-may-be-null}
+  If you supply the
+  \code{-Astubs=collection-object-parameters-may-be-null.astub}
```

Comment:
Because `collection-object-parameters-may-be-null.astub` is in the same directory as the Nullness Checker, it will automatically be found. You only need to add `checker.jar/` is the stub file is in the top level directory of the jar.  (Also, good to ask, though.)

Reply:
`javadoc.astub` is at the top level because it has annotations for multiple type systems.